.github/workflows/lint-test.yaml

```diff
@@ -14,7 +14,7 @@ jobs:
 
       - name: Run chart-testing (lint)
         id: lint
-        uses: helm/chart-testing-action@v1.0.0
+        uses: quay.io/helmpack/chart-testing:v3.3.0
```

Comment:
This does not work. We have to use the chart-testing-action. Internally it's using the chart-testing image. If we want to specify a specific version then we need to do that via parameters.

Reply:
@torstenwalter If we're going to migrate to helm3 shouldn't we start using [chart-testing-action@v3.0.0](https://github.com/marketplace/actions/helm-chart-testing-helm-3#example-workflow)should I update it?